sandboxes/rslib-react-component/stories/mfCounter.stories.tsx

```diff
@@ -0,0 +1,10 @@
+// @ts-ignore
```

Comment:
Is it common to add `@ts-ignore` for remote modules?

Reply:
i write the reason in note